internal/data/spicedb.go

```diff
@@ -687,6 +682,11 @@ func (s *SpiceDbRepository) determineConsistency(consistency *apiV1beta1.Consist
 		}
 	}
 
```

Comment:
Even though I don't think it's worth renaming things (i.e. `fullyConsistent` to something like `fullyConsistentAsDefault`) here and in all the places, since relations-api is effectively deprecated, I think a comment here explaining the new intent behind the flag, maybe along with a TODO to rename, would be good.